it/controllers/AddressLookupControllerISpec.scala

```diff
@@ -1,269 +1,257 @@
 package controllers
 
-import java.util.UUID
-
 import itutil.IntegrationSpecBase
-import model.{ConfirmableAddress, ConfirmableAddressDetails, JourneyConfig, JourneyData}
-import org.jsoup.Jsoup
+import itutil.config.IntegrationTestConstants._
+import itutil.config.PageElementConstants._
+import model.{ConfirmableAddress, ConfirmableAddressDetails}
 import play.api.http.HeaderNames
+import play.api.http.Status._
 import play.api.libs.json.{JsObject, Json}
 import play.api.test.FakeApplication
 import uk.gov.hmrc.address.v2.Country
 
 class AddressLookupControllerISpec extends IntegrationSpecBase {
-  override implicit lazy val app = FakeApplication(additionalConfiguration = fakeConfig())
 
-  val testConfig = JourneyData(JourneyConfig(continueUrl = "A url"))
-  val csrfToken = () => UUID.randomUUID().toString
+  override implicit lazy val app = FakeApplication(additionalConfiguration = fakeConfig())
 
   "The lookup page" should {
     "pre-pop the postcode and filter on the view when they are passed in as query parameters and drop selected address on load" in {
+      stubKeystore(testJourneyId, testConfigDefaultAsJson, OK)
+      stubKeystoreSave(testJourneyId, testConfigWithoutAddressAsJson, OK)
 
-      stubKeystore("Jid123", Json.toJson(testConfig).as[JsObject], 200)
-      stubKeystoreSave("Jid123", Json.toJson(testConfig.copy(selectedAddress = None)).as[JsObject], 200)
-      val sessionCookie = getSessionCookie(Map("csrfToken" -> csrfToken()))
-      val fResponse = buildClient("/lookup?postcode=AB11+1AB&filter=bar").
-        withHeaders(HeaderNames.COOKIE -> sessionCookie, "Csrf-Token" -> "nocheck").
-        get()
+      val sessionCookie = sessionCookieWithCSRF
```

Comment:
Does this need to be a separate val? Can you not call the method directly in the line below?